packages/components/src/Countdown/Countdown.tsx

```diff
@@ -12,10 +11,10 @@ type GranularityOptions = "dhms" | "hms" | "ms" | "s" | "dhm" | "dh" | "d";
 interface CountdownProps {
   /**
    * The date that is being counted down to
-   * Civil Time of time is to be displayed.
+   * time is to be displayed.
```

Comment:
I think the wording on this docstring could be improved

Reply:
Addressed.